gateway/1.1/modules/ROOT/pages/_partials/forward-proxy-config.adoc

```diff
@@ -0,0 +1,104 @@
+//tag::configuration[]
```

Comment:
Is this for the 1.1 or 1.4 version? Because forward proxy is available only from 1.4